src/client/components/CommentFooter/CommentFooter.less

```diff
@@ -15,7 +15,21 @@
     font-size: 14px;
     margin: 0 8px;
   }
+  
+  &__bullet_br:after {
+      content: "\2022";
```

Comment:
looks like some spacing issues here, please align appropriately

Reply:
It's modifier so we can put it inside `&__bullet`.